ambry-tools/src/main/java/com.github.ambry/tools/admin/BlobValidator.java

```diff
@@ -368,38 +373,38 @@ private void validateBlobOnAllReplicas(BlobId blobId, ClusterMap clusterMap, boo
       }
     }
     compareListOfBlobContent(blobId.getID(), replicaIdBlobContentMap);
-    logger.info("Summary ");
+    boolean errorLoggingRequired = false;
+    if (responseMap.keySet().size() != 1 || !replicaIdBlobContentMap.keySet().contains(ServerErrorCode.No_Error)) {
+      errorLoggingRequired = true;
+    }
+    if (errorLoggingRequired) {
+      logger.error("Summary of responses for " + blobId);
+    } else {
+      logger.debug("Summary of responses for " + blobId);
+    }
     for (ServerErrorCode serverErrorCode : responseMap.keySet()) {
-      logger.info(serverErrorCode + ": " + responseMap.get(serverErrorCode));
+      if (errorLoggingRequired) {
+        logger.error(serverErrorCode + ": " + responseMap.get(serverErrorCode));
```

Comment:
trying to understand why this variable `errorLoggingRequired` is required. Why not print this in debug and someone who wants to view this goes to DEBUG level?